ui/src/shared/utils/downloadTimeseriesCSV.ts

```diff
@@ -50,7 +60,7 @@ const timeseriesToCSV = async (
     table[i][timeIndex] = new Date(table[i][timeIndex]).toISOString()
   }
 
-  return table
+  return unparse(table)
```

Comment:
what does `unparse` mean? serialize?

Reply:
yeah. unfortunate name, but it's directly from the papa parse lib we use.